pom.xml

```diff
@@ -1,56 +1,62 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
Just as an observation, we should think about some common rules regarding the formatting. Otherwise we will always have a lot of unwanted diffs in the pull requests. However, not sure if you can set-up rules, just for one project in IntelliJ/Eclipse.

Reply:
This is probably code formatting because of Eclipse/IntelliJ differences.